src/util/time-util.ts

```diff
@@ -9,7 +9,14 @@ export function timeToString(time: Time): string | null {
 }
 
 export function dateToString(date: Moment): string | null {
-  return date ? date.format("YYYY-MM-DD") : null;
+  var ret: string;
+  if (date) {
+    const year = moment().format('YYYY');
+    ret = date.add(year, 'years').format(year + "-MM-DD");
+  } else {
+    ret = null;
+  }
+  return ret;
```

Comment:
```suggestion
  if (date) {
    const year = moment().format('YYYY');
    return date.add(year, 'years').format(year + "-MM-DD");
  }
  return null;
```

Reply:
Aren't these two the same?
```ts
return date.add(year, 'years').format(year + "-MM-DD");
```

```ts
return date.format(year + "-MM-DD");
```

Why do we add a year first and then overwrite it? If this is necessary, please a short comment.

Reply:
Ah sorry I did something with the add and it wasn't working so I used format without removing the add